include/ignition/sensors/ImuSensor.hh

```diff
@@ -35,6 +35,27 @@ namespace ignition
   {
     // Inline bracket to help doxygen filtering.
     inline namespace IGNITION_SENSORS_VERSION_NAMESPACE {
+
+    // \brief Reference frames enum
```

Comment:
please use 3 slashes `///` for all the doc-strings in this class to ensure that Doxygen will parse it

Reply:
Done